### PR TITLE
Feature - Restore foreign key

### PR DESCRIPTION
**Pull request by lyazii22**
- [ ] Remove interleave and convert back to foreign key implementation.

### CODE REVIEW

File: ui/src/app/components/object-detail/object-detail.component.html [hunk 1]
```diff
@@ -753,7 +742,7 @@ <h3 class="title">
           "
           mat-raised-button
           color="primary"
```
Comment (shreyakhajanchi) [inline ui/src/app/components/object-detail/object-detail.component.html:744]: Modify the SQL in SQL tab after interleaving property is removed

File: ui/src/app/components/object-detail/object-detail.component.html [hunk 2]
```diff
@@ -700,17 +700,6 @@ <h3 class="title">
...
-                      interleaveStatus.tableInterleaveStatus &&
-                      !interleaveStatus.tableInterleaveStatus.Possible
-                    "
-                    [disabled]="element.get('spName').value === ''"
-                    mat-menu-item
-                    (click)="convertToFk()"
-                  >
-                    <span>Convert to Foreign Key</span>
-                  </button>
                 </mat-menu>
```
Comment (shreyakhajanchi) [inline ui/src/app/components/object-detail/object-detail.component.html:703]: In case of two foreign keys, after interleaving one the second foreign key appears non adjacent to the source one: https://drive.google.com/file/d/19YkXKfZCM-BLLh_EKM-GXsK3z9astJ52/view?usp=share_link&resourcekey=0-oQLq3JDk0YtDSc8dSXz1Lg

File: ui/src/styles.scss [hunk 3]
```diff
@@ -53,13 +53,18 @@ body {
...
 
 .success {
   color: green;
 }
 
+.suggestion {
+  color: #f6be00;
+  transform: rotate(180deg);
+}
+
```
Comment (shreyakhajanchi) [inline ui/src/styles.scss:67]: these css changes do not seem to be part of this PR same for restore index PR

Comment (Ronak-Searce) [reply ui/src/styles.scss]: Updated

File: ui/src/app/components/object-detail/object-detail.component.html (hunk 1 above)
Comment (Ronak-Searce) [reply ui/src/app/components/object-detail/object-detail.component.html:775]: Updated

**Review by shreyakhajanchi — APPROVED** [review]
LGTM
